packages/prettier-config/index.json

```diff
@@ -0,0 +1,6 @@
+{
+  "printWidth": 79,
+  "singleQuote": true,
+  "trailingComma": "all",
+  "quoteProps": "preserve"
```

Comment:
?

Reply:
it's a new setting.https://prettier.io/blog/2019/04/12/1.17.0.html#add-an-option-to-modify-when-prettier-quotes-object-properties-5934-by-azz wasn't sure what to set it too, there are a few places where we intentionally disable prettier to unnecessarily quote `__DEV__` in `eslintrc.js`'s otherwise dev expression does something (idk) so that was why i went with this option

Reply:
i'd say if we were non going to preserve it we should go with the 'consistent' setting i always hated:

```
alias: {
  react: ...,
  'react-dom': ...
}
```

Reply:
if you want. both `as-needed` and `consistent` are fine by me